automation/travisci-test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
```

Comment:
I'm still not sure if we want `-e` active here? As I see it (at least when the script is declared inline) in `.travis.yaml` it appears that as soon as the script fails, the output capturing seems to get stopped. (Example: https://travis-ci.org/github/kubevirt/kubevirt/builds/693857809)
I suspect that the recording is done async, therefore some of the output is missing. @EdDev did you try and see what happens if the complete output is here when i.e. `make generate` leaves changes in the git dir (due to formating being wrong) and the build therefore fails?

Reply:
Maybe you have missed the source of this: https://github.com/kubevirt/kubevirt/pull/3413#issuecomment-636097806

The cuts at the moment is the exact problem this one comes to fix (except the obvious of not having a mess in the travis yaml).